statvar_imports/oecd/quarterly_gdp/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "import_specifications": [
+    {
+      "import_name": "OECDQuarterlyGDP",
+      "import_type": "statvar_processor",
+      "gcs_bucket": "datcom-prod-imports",
+      "curator_emails": [
+        "support@datacommons.org"
+      ],
+      "provenance_url": "https://data.oecd.org/gdp/quarterly-gdp.htm",
+      "provenance_description": "Quarterly GDP data from the Organisation for Economic Co-operation and Development (OECD). This data is retrieved using the SDMX API.",
+      "scripts": [
+        "download.sh",
+        "process.sh"
+      ],
+      "source_files": [
+        "input/*"
+      ],
+      "import_inputs": [
+        {
+          "template_mcf": "output/oecd_quarterly_gdp.tmcf",
+          "node_mcf": "output/*.mcf",
+          "cleaned_csv": "output/oecd_quarterly_gdp.csv",
+          "statvar_counters_csv": "oecd_quarterly_gdp/validation/statvar_counters.csv"
```

Comment:
Pls don't keep the counters csv as import input. It would get copied into the same gcs folder that has the tmcf but that tmcf won't apply to this file.

Just creating the file in the va;idtion folder should be sufficient for it to be copied to the GCS validation folder. Let us know if that isn't working.